azurerm/resource_arm_kubernetes_cluster_test.go

```diff
@@ -357,7 +349,7 @@ resource "azurerm_kubernetes_cluster" "test" {
     client_secret = "%s"
   }
 }
-`, rInt, location, rInt, rInt, rInt, clientId, clientSecret)
+`, rInt, location, rInt, rInt, clientId, clientSecret)
```

Comment:
can you add a separate test case (and import test) for this which includes the linux profile block e.g. `TestAccAzureRMKubernetesCluster_linuxProfile`